### ISSUE TITLE
The Issue only occurs in the aan dataset

### ISSUE THREAD

**Issue by maisuiqianxun** [HazyResearch/state-spaces#124]
Here is the related info:
"Global seed set to 1112
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
`Trainer(limit_train_batches=1.0)` was configured so 100% of the batches per epoch will be used..
`Trainer(limit_val_batches=1.0)` was configured so 100% of the batches will be used..
[2023-10-26 17:33:52,297][__main__][INFO] - Instantiating callback <src.callbacks.timer.Timer>
[2023-10-26 17:33:52,298][__main__][INFO] - Instantiating callback <src.callbacks.params.ParamsLog>
[2023-10-26 17:33:52,298][__main__][INFO] - Instantiating callback <pytorch_lightning.callbacks.ModelCheckpoint>
[2023-10-26 17:33:52,299][__main__][INFO] - Instantiating callback <pytorch_lightning.callbacks.RichModelSummary>
[2023-10-26 17:33:52,299][__main__][INFO] - Instantiating callback <pytorch_lightning.callbacks.RichProgressBar>
[2023-10-26 17:33:53,258][datasets.builder][WARNING] - Found cached dataset csv (/home/user/.cache/huggingface/datasets/csv/default-b6b89ff0e0de910a/0.0.0/6954658bab30a358235fa864b05cf819af0e179325c740e4bc853bcc7ec513e1)
100%|██████████| 3/3 [00:07<00:00,  2.34s/it]
Map (num_proc=4): 100%|██████████| 147086/147086 [00:56<00:00, 1560.50 examples/s]"

The code of S4 has been in the data preprocessing stage for a long time and cannot run to the training stage, which is only occurred in the aan dataset. Does anybody know how to solve this issue?

**Comment by maisuiqianxun** [comment]
This may be caused by the excessive memory usage.
